stubs/PyYAML/yaml/__init__.pyi

```diff
@@ -7,6 +7,7 @@ from . import resolver as resolver  # Help mypy a bit; this is implied by loader
 from .constructor import BaseConstructor
 from .cyaml import *
 from .dumper import *
+from .dumper import _Inf
```

Comment:
I'm not sure what the best place to define the alias is.. this feels a little awkward.

Reply:
I think this is fine!